andela/mrm_api
# #164858926 Create a database model for office structure

### What does this PR do?
Adds office_structures table

### Description of the task to be completed?
Create a new office structure model

### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/andela/mrm_api.git`
2. Cd into the cloned repo 
3. Git pull branch ft-office-structure-164858926
4. Run migrations
5. New table - office_structures should be created in the database

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#164858926](https://www.pivotaltracker.com/story/show/164858926)
### Checklist:
